### PR TITLE
Changed PXEngine references to Pixate.

### PR DESCRIPTION
**Pull request by bobspace**

### CODE REVIEW

**Comment by thelonious** [comment]
Hey Bob,

This line "<module platform="iphone">com.pixate.pxengine</module>" needs to remain as it is. We're keeping the module ID so we don't break existing Titanium projects.

-Kevin

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Unzip the module into your project's modules directory or install it globally. S
 
 You can use the Ti Studio interface to add the Pixate Engine module into your project, or just edit the tiapp.xml file directly and add this to your <modules> section:
 
-	<module platform="iphone">com.pixate.pxengine</module>
```
Comment (thelonious) [inline README.md:15]: We need to keep this line as is so we don't break existing Titanium projects

**Comment by bobspace** [comment]
Is that better?

**Comment by thelonious** [comment]
parfait!

**Pull request merged**